### ISSUE TITLE
"Cannot find collection's corresponding offer" error when throughput is provisioned on database level

### ISSUE THREAD

**Issue by vchalmel** [Azure/azure-cosmosdb-spark#284]
When using the cosmosdb option to define max throughput at database scale without specifically provisionning throughput when creating the collection, writing in the collection fails with the error "Cannot find collection's corresponding offer".

Quickly searching in the source for this error message, it seems to be caused by the call of a "replaceOffer" function used to temporarily increase the provisionned throughput for the target collection.

**Comment by sagarsharmas** [comment]
I'm getting the same issue when throughput is provisioned on database level.
getCollectionThroughput function is called while writing to CosmosDB which has the following definition:
```
def getCollectionThroughput: Int = {
    var offers = documentClient.queryOffers(s"SELECT * FROM c where c.offerResourceId = '${getCollection.getResourceId}'", null).getQueryIterable.toList
    if (offers.isEmpty) {
      offers = documentClient.queryOffers(s"SELECT * FROM c where c.offerResourceId = '${getDatabase.getResourceId}'", null).getQueryIterable.toList
      // database throughput
      if (offers.isEmpty) {
          throw new IllegalStateException("Cannot find the collection corresponding offer.")
      }
    }
```
Basically, `offers = documentClient.queryOffers(s"SELECT * FROM c where c.offerResourceId = '${getDatabase.getResourceId}'", null).getQueryIterable.toList ` is failing.

**Comment by nomiero** [comment]
This is fixed in 1.3.3 version, please upgrade to that.
